### DISCUSSION TITLE
HLS Dump with API doesn't seem to work

### DISCUSSION THREAD

**Discussion by ArturLinnik** [OvenMediaLabs/OvenMediaEngine#1265] [Help] [answered]
I don't know why but it isn't working for me.

When I configure the LLHLS dumping it works right. The config is as follows: 

```
            <LLHLS>
              <ChunkDuration>0.5</ChunkDuration>
              <SegmentDuration>6</SegmentDuration>
              <SegmentCount>10</SegmentCount>
              <CrossDomains>
                <Url>*</Url>
              </CrossDomains>

             <Dumps>
                <Dump>
                  <Enable>true</Enable>
                  <TargetStreamName>*</TargetStreamName>

                  <Playlists>
                    <Playlist>llhls.m3u8</Playlist>
                    <Playlist>abr.m3u8</Playlist>
                  </Playlists>
                  <OutputPath>/vods/${VHostName}_${AppName}_${StreamName}/${ISO8601}</OutputPath>
                </Dump>
              </Dumps>

            </LLHLS>
```
I have tried to keep these fields (although it dumps automatically my streamings) and also removing them but when I try to dump it with the API with the following body:

```
{
        "outputStreamName": output_stream_name,
        "id": stream_id,
        "outputPath": "/root/ome/vods/",
    }
```
It doesn't create anything.

I have already checked that OME has permissions for writing in that directory because the automatic dumping works and saves the files there.

The output of the API request is:

```
{
  "message": "OK",
  "statusCode": 200
}
``` 
which btw is strange because it should be as follows, according to the documentation:

```
{
	"statusCode": 200,
	"message": "OK",
	"response": [
		"stream",
		"stream2"
	]
}
```

And the response field doesn't appear although I am starting the dumping of the stream multiple times with different ids.

I am sure I'm doing something wrong but I don't know what and the OME's logs don't help me.

I would be very grateful if you could help me with this.

**Comment by getroot** [comment]
Please upload the full OME log file and provide the full text of your request json.

**Comment by getroot** [comment, marked as answer]
The full log file is required to accurately analyze the problem. Please upload the entire ovenmediaengine.log file. What version of OME are you using? It doesn't appear to be the latest version.

"id" is of string type. Entering 1111 will cause the request to fail.

No files in your /root/ome/vods?

What does stop after 5 seconds mean specifically? Does OME crash?

**Reply by ArturLinnik** [reply]
Ok, it was a rookie mistake, I'm sorry

I'm launching OME with Docker and I had a mistake defining the volumes. It was all working well all the time, I figured it out when I tried to access the ovenmediaengine.log file and I couldn't find it because it obviously wasn't in my machine but inside the container.
